pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/plan/ServerRequestPlanVisitor.java

```diff
@@ -104,7 +104,7 @@ public static ServerPlanRequestContext build(MailboxService<TransferableBlock> m
     pinotQuery.setExplain(false);
     ServerPlanRequestContext context =
         new ServerPlanRequestContext(mailboxService, requestId, stagePlan.getStageId(), timeoutMs,
-            stagePlan.getServerInstance().getHostname(), stagePlan.getServerInstance().getPort(),
+            stagePlan.getServerInstance().getHostname(), stagePlan.getServerInstance().getQueryMailboxPort(),
```

Comment:
@walterddr I think this is fixed in #10157

Reply:
:-D

Reply:
sweet will close mine then.